packages/cli/commands/cms/convertFields.js

```diff
@@ -15,7 +15,7 @@ const { trackConvertFieldsUsage } = require('../../lib/usageTracking');
 const { logErrorInstance } = require('../../lib/errorHandlers/standardErrors');
 const i18nKey = 'cli.commands.convertFields';
 
-exports.command = 'convert-fields';
+exports.command = 'convert-fields [src] [--fieldOptions]';
```

Comment:
`src` is an optional argument so we shouldn't list it here. We should also change it to be an option instead of a positional since we are preferring the use of options over positionals (and that way this change won't be a breaking change, b/c currently the command does not support passing in src as a positional).

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1008/commits/cd2468ad58eea69e033939ec6fe39a472fe25da8